src/Tests/Contract/PaymentPayActionHelperTrait.php

```diff
@@ -28,4 +28,19 @@ protected static function createPaymentPayAction(ShopInterface $shop, array $req
             $requestData,
         );
     }
+
+    private function createPaymentPayActionForApplication(IdsCollection $orderIds, ShopInterface $shop, array $requestData = []): PaymentPayAction
```

Comment:
isn't that the same as `createPaymentPayAction`?

Reply:
s.a., I call the other createOrder here